src/patternfly/components/Tabs/tabs.scss

```diff
@@ -1,106 +1,254 @@
+
 .pf-c-tabs {
-  // Primary tab items
-  --pf-c-tabs__item--BackgroundColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-tabs__item--BorderColor: var(--pf-global--BorderColor--100);
-  --pf-c-tabs__item--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-tabs__item--m-current--ZIndex: var(--pf-global--ZIndex--sm);
-  --pf-c-tabs__item--m-current--Color: var(--pf-global--active-color--100);
-  --pf-c-tabs__item--m-current--BorderTopWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-tabs--Inset: 0;
+  --pf-c-tabs--before--BorderStyle: solid;
+  --pf-c-tabs--before--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-tabs--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-tabs--before--BorderTopWidth: 0;
+  --pf-c-tabs--before--BorderRightWidth: 0;
+  --pf-c-tabs--before--BorderBottomWidth: var(--pf-c-tabs--before--BorderWidth);
+  --pf-c-tabs--before--BorderLeftWidth: 0;
+  --pf-c-tabs--m-vertical--Inset: var(--pf-global--spacer--lg);
+  --pf-c-tabs--m-vertical--MaxWidth: #{pf-size-prem(250px)};
+  --pf-c-tabs--m-vertical--m-box--Inset: var(--pf-global--spacer--xl);
 
-  // Secondary tab items
-  --pf-c-tabs__item--hover--Color: var(--pf-global--Color--dark-200);
 
   // Tab buttons
-  --pf-c-tabs__button--Color: var(--pf-global--Color--100);
-  --pf-c-tabs__button--FontWeight: var(--pf-global--FontWeight--normal);
-  --pf-c-tabs__button--Background: transparent;
-  --pf-c-tabs__button--OutlineOffset: calc(-1 * var(--pf-global--spacer--xs));
+  --pf-c-tabs__button--Color: var(--pf-global--Color--200);
+  --pf-c-tabs__button--BackgroundColor: transparent;
+  --pf-c-tabs__button--OutlineOffset: calc(-1 * #{pf-size-prem(6px)});
   --pf-c-tabs__button--PaddingTop: var(--pf-global--spacer--sm);
-  --pf-c-tabs__button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-tabs__button--PaddingBottom: var(--pf-global--spacer--sm);
-  --pf-c-tabs__button--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-tabs__button--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-tabs__button--md--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-tabs__button--md--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-tabs__item--m-current__button--Color: var(--pf-global--Color--100);
+  --pf-c-tabs__item--m-current__button--Background: var(--pf-global--BackgroundColor--100);
+  --pf-c-tabs--m-vertical__button--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-tabs--m-vertical__button--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-tabs--m-box__button--BackgroundColor: var(--pf-global--BackgroundColor--200);
+
+
+  // Button before
+  --pf-c-tabs__button--before--BorderStyle: solid;
+  --pf-c-tabs__button--before--BorderColor: var(--pf-c-tabs--before--BorderColor);
+  --pf-c-tabs__button--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-tabs__button--before--BorderTopWidth: 0;
+  --pf-c-tabs__button--before--BorderRightWidth: 0;
```

Comment:
are these vars necessary?

Reply:
Yes, they're updating in the vertical variant.